modules/salt/pages/salt-gitfs.adoc

```diff
@@ -0,0 +1,65 @@
+[[salt.gitfs]]
+= The ``gitfs`` Fileserver Backend
+
+In {productname}, [package]``pygit2`` is the supported Python interface to git.
+When [package]``pygit2`` is installed the ``gitfs`` fileserver backend is available and it is a supported feature.
+
+
+Configuration options are set in the [path]``/etc/salt/master`` file or a separate configuration file [path]``/etc/salt/master.d/``.  Basic settings are:
+
+
+fileserver_backend::
+List of fileserver backends that the Salt master checks for files in the order they are defined.  Options:
++
+* [literal]``roots``: Files local on the Salt master ({productname} Server).
+{suse} strongly recommends to prefer [literal]``roots`` (local files) over [literal]``gitfs``.
```

Comment:
keeping `roots` is required to keep the product running. You can only "add" gitfs in addition. 
We should make it clear that it is not allowed to remove `roots`